### PR TITLE
A couple fixes for modules built with kernel 5.10

### PR DESCRIPTION
**Pull request by jmarcet**
Built and run tested on master x86-64.

### CODE REVIEW

File: package/kernel/linux/modules/usb.mk [hunk 1]
```diff
@@ -1216,6 +1216,7 @@ $(eval $(call KernelPackage,usb-net-mcs7830))
 
 define KernelPackage/usb-net-smsc95xx
   TITLE:=SMSC LAN95XX based USB 2.0 10/100 ethernet devices
+  DEPENDS:=+LINUX_5_10:kmod-libphy
```
Comment (abajk) [inline package/kernel/linux/modules/usb.mk:1219]: This fixes kmod-usb-net-smsc95xx. You should change commit title.

Comment (abajk) [inline package/kernel/linux/modules/usb.mk:1219]: This fixes kmod-usb-net-smsc95xx. You should change commit title.

This commit is similar to the one in my [PR.](https://github.com/openwrt/openwrt/pull/4002) I missed this PR due to incorrect commit title.

**Review by abajk — CHANGES_REQUESTED** [review]

**Comment by guidosarducci** [comment]
@jmarcet @abajk I've also been working a few days on a branch with several fixes following up after #3989 , including two similar to yours: https://github.com/openwrt/openwrt/pull/3989#issuecomment-801331066. You both should have been notified since  you're mentioned in that PR.

I'm writing the PR for the new branch now. If you could have a look, and we come to consensus on the various fixes, let's coordinate what goes in which PR.

**Comment by hauke** [comment]
Similar changes were pushed to master, see a374b8f190518a74dcc99d6bf3cf53adb7511963 and ad11fc8c236fca5283ed500d87504543f2768312. Please check if your changes are still needed and rebase on top of these changes.

**Comment by adschm** [comment]
@jmarcet Ping.

**Comment by adschm** [comment]
Closed due to inactivity.

**Pull request closed without merging**